event/oracle/expected.go

```diff
@@ -0,0 +1,18 @@
+package oracle
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/medibloc/panacea-doracle/config"
+	"github.com/medibloc/panacea-doracle/panacea"
+	"github.com/medibloc/panacea-doracle/sgx"
+)
+
+// reactor contains all ingredients needed for handling this type of event
+type reactor interface {
```

Comment:
why don't we share this `reactor` interface for all events?

Reply:
I didn't understand the question correctly.
This reactor is declared as a variable in every event.

Reply:
It is in package `oracle`. I meant to make this interface to be shared to other types of events, for example data verification. 
There is a similar interface defined in #80 as well (https://github.com/medibloc/panacea-doracle/blob/b2fa22dc5d5de2bfd5b394ffab29459ea74420ac/event/datadeal/data_verification_event.go#L32).

Reply:
Oh, I understand.
The event was declared in datadeal as well.
I will refactor.
Thanks. :)